docs/Functions.md

```diff
@@ -0,0 +1,59 @@
+## Functions
+*This file is automatically generated from the
+            [def files](/onnx/defs) via [this script](/onnx/defs/gen_doc.py).
+            Do not modify directly and instead edit function definitions.*
+## ai.onnx (default)
+  * <sub>experimental</sub><a href="#FuncMeanVarianceNormalization">FuncMeanVarianceNormalization</a>
+
+
+
+### <sub>experimental</sub> <a name="FuncMeanVarianceNormalization"></a><a name="funcmeanvariancenormalization">**FuncMeanVarianceNormalization**</a>
+
+  A MeanVarianceNormalization Function: Perform mean variance normalization on the input tensor X using formula: <br/> ``` (X-EX)/sqrt(E(X-EX)^2) ``` <br/>INPUT: X(float/float16/double) with Shape [N,C,W,H] <br/>ATTRIBUTE: <br/>&nbsp;&nbsp;&nbsp;&nbsp;axes: will be passed to ReducedMean Ops. Use [0,2,3] for no across channel, [0,1,2,3] for across channel caculation.<br/>&nbsp;&nbsp;&nbsp;&nbsp;(The KeepDims attribute in ReducedMean ops is set to true for caculation)<br/>OUTPUT: X(float/float16/double) with Shape [N,C,W,H] <br/>
+
+#### Version
+
+This version of the function has been available since version 8 of the default ONNX operator set.
+
+#### Inputs
+
+<dl>
+<dt>X; </dt>
+<br/></dl>
+
+#### Outputs
+
+<dl>
+<dt>X_MVN; </dt>
+<br/></dl>
+
+#### Attributes
+
+<dl>
+<dt>axes;<br/></dt>
+</dl>
+
+#### Nodes
```

Comment:
Let's figure out a better way to show function body. 

We may add a netron picture or link in function description (we may ask Netron to be able to visualize a function).

Reply:
I guess this can be next step as we prob need to explore the ways to do that